src/interfaces/decorator-return.ts

```diff
@@ -0,0 +1,3 @@
+export type MethodDecoratorReturn = (target: any, propertyKey: string, descriptor: PropertyDescriptor) => void;
```

Comment:
I don't yet understand where these signatures come from. Could you explain that?